homework3/LZW.Algorithm/LZW.Algorithm/Trie.cs

```diff
@@ -0,0 +1,79 @@
+// <copyright file="Trie.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace LZW.Algorithm;
+
+/// <summary>
+/// This trie.
+/// </summary>
+public class Trie
+{
+      private readonly TrieNode root;
+      private int nextCode = 0;
+
+      /// <summary>
+      /// Initializes a new instance of the <see cref="Trie"/> class.
+      /// </summary>
+      public Trie()
+      {
+            this.root = new TrieNode(this.nextCode++);
+      }
```

Comment:
```suggestion
      public Trie()
          => this.root = new TrieNode(this.nextCode++);
```
Тем более что тут отступ лишний. Отступы в четыре пробела по стайлгайду.